diff --git a/Makefile b/Makefile
index 2b98cac..ec07d6a 100644
--- a/Makefile
+++ b/Makefile
@@ -82,7 +82,7 @@ $(LUACLIB_PATH):
 	mkdir $(LUACLIB_PATH)

 $(LUACLIB_PATH)/sys.so: $(addprefix $(LIB_PATH)/, $(LIB_SRC)) | $(LUACLIB_PATH)
-	$(CC) $(CCFLAG) $(INCLUDE) -o $@ $^ $(SHARED)
+	$(CC) $(CCFLAG) $(INCLUDE) -o $@ $^ $(SHARED) -DUSE_OPENSSL -lssl -lcrypt
 $(LUACLIB_PATH)/zproto.so: lualib-src/zproto/lzproto.c lualib-src/zproto/zproto.c | $(LUACLIB_PATH)
 	$(CC) $(CCFLAG) $(INCLUDE) -o $@ $^ $(SHARED)
 $(LUACLIB_PATH)/test.so: $(LIB_PATH)/lualib-test.c | $(LUACLIB_PATH)
