diff --git a/Makefile b/Makefile
index c406837..f4a518e 100644
--- a/Makefile
+++ b/Makefile
@@ -82,19 +82,19 @@ all: \
 	$(LUACLIB_PATH)/test.so \
 
 $(TARGET):$(OBJS) $(LUA_STATICLIB) $(MALLOC_STATICLIB)
 	$(LD) -o $@ $^ $(LDFLAG)
 
 $(LUACLIB_PATH):
 	mkdir $(LUACLIB_PATH)
 
 $(LUACLIB_PATH)/sys.so: $(addprefix $(LIB_PATH)/, $(LIB_SRC)) | $(LUACLIB_PATH)
-	$(CC) $(CCFLAG) $(INCLUDE) -o $@ $^ $(SHARED)
+	$(CC) $(CCFLAG) $(INCLUDE) -o $@ $^ $(SHARED) -DUSE_OPENSSL -lssl -lcrypto
 $(LUACLIB_PATH)/zproto.so: $(LIB_PATH)/zproto/lzproto.c $(LIB_PATH)/zproto/zproto.c | $(LUACLIB_PATH)
 	$(CC) $(CCFLAG) $(INCLUDE) -o $@ $^ $(SHARED)
 $(LUACLIB_PATH)/http2.so: $(LIB_PATH)/lualib-http2.c | $(LUACLIB_PATH)
 	$(CC) $(CCFLAG) $(INCLUDE) -o $@ $^ $(SHARED)
 $(LUACLIB_PATH)/test.so: $(LIB_PATH)/lualib-test.c | $(LUACLIB_PATH)
 	$(CC) $(CCFLAG) $(INCLUDE) -o $@ $^ $(SHARED)
 
 .depend:
 	@$(CC) $(INCLUDE) -MM $(SRC) 2>/dev/null |\
